src/Controller/AdminController.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Controller;
+
+class AdminController extends AbstractController
+{
+    public function course(): string
+    {
+        return $this->twig->render('Admin/course.html.twig');
```

Comment:
so, no form validations yet?

Reply:
Yes it's done now!
I change for not use service.
your correction will be easier